database/src/main/resources/db/migration/V1_25__Add_gas_fee_volumes.sql

```diff
@@ -0,0 +1,98 @@
+-- Host future and previous migrated data
+CREATE TABLE IF NOT EXISTS tx_gas_cache
+(
+    id           BIGSERIAL PRIMARY KEY,
+    hash         VARCHAR(64) UNIQUE    NOT NULL,
+    tx_timestamp TIMESTAMP             NOT NULL,
+    gas_wanted   INT                   NOT NULL,
+    gas_used     INT                   NOT NULL,
+    fee_amount   DOUBLE PRECISION      NOT NULL,
+    processed    BOOLEAN DEFAULT false NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS tx_gas_cache_tx_timestamp_idx ON tx_gas_cache (tx_timestamp);
+CREATE INDEX IF NOT EXISTS tx_gas_cache_tx_gas_used_idx ON tx_gas_cache (gas_used);
+CREATE INDEX IF NOT EXISTS tx_gas_cache_tx_processed_idx ON tx_gas_cache (processed);
+
+-- Aggregate daily gas volume
+CREATE TABLE IF NOT EXISTS tx_gas_fee_volume_day
+(
+    tx_timestamp TIMESTAMP PRIMARY KEY,
+    gas_used     BIGINT NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS tx_gas_fee_volume_day_gas_used ON tx_gas_fee_volume_day (gas_used);
+
+-- Aggregate hourly gas volume
+CREATE TABLE IF NOT EXISTS tx_gas_fee_volume_hour
+(
+    tx_timestamp TIMESTAMP PRIMARY KEY,
+    gas_used     BIGINT NOT NULL
+);
+
+CREATE INDEX IF NOT EXISTS tx_gas_fee_volume_hour_gas_used ON tx_gas_fee_volume_hour (gas_used);
+
+-- Host future and previous migrated data
+INSERT
+INTO tx_gas_cache(hash, tx_timestamp, gas_wanted, gas_used, fee_amount)
+SELECT DISTINCT hash,
+                tx_timestamp,
+                gas_wanted,
+                gas_used,
+                (
+                    SELECT min_gas_fee
```

Comment:
This is not the value we want here. We want the fee amount paid in the tx. This will require digging into the jsonb field to get to it.